etc/integration-tests/src/nearcore_utils.rs

```diff
@@ -113,3 +119,50 @@ pub async fn start_neard(
 
     Ok(child)
 }
+
+pub fn neard_path() -> anyhow::Result<PathBuf> {
+    std::env::current_dir()
```

Comment:
I wonder if instead of the current directory we should use `refiner_utils::get_repository_root()`